frontend/components/text-classifier/labeling-rules/RuleDefinition.vue

```diff
@@ -54,10 +69,18 @@ export default {
     query() {
       return this.dataset.query.text;
     },
+    activeLabel() {
+      return this.selectedLabels
+        ? this.selectedLabels[0]
```

Comment:
`selectedLabel` is updated as the label value in `updateLabel` method. Should be considered still as an array ?

Reply:
I think yes, it should be treated as an array with a single element, maybe in the future with multiple elements.

I changed label => labels